src/pages/api/prayer-times.ics.ts

```diff
@@ -32,10 +32,12 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
             .toDate(),
           summary: name,
         });
-        event.createAlarm({
-          type: ICalAlarmType.audio,
-          triggerBefore: (alarm ? +alarm : 15) * 60,
-        });
+        if (+alarm !== 0) {
```

Comment:
Still need to check if the alarm not undefined 
`If (alarm && +alarm > 0)`

Reply:
Let's try that :crossed_fingers: 
```suggestion
        if (alarm && +alarm > 0) {
```